### PR TITLE
fix non-arc, cannot create __weak reference

### PR DESCRIPTION
**Pull request by sanhue0302**
i am sorry about i don't know how to modify old pull request, 
so i create a new pull request again.

### CODE REVIEW

**Comment by thomasvl** [comment]
To update a pull request, just update your branch on github, it will then see the updates.

**Comment by thomasvl** [comment]
This seems to have multiple changes not related to the __weak usage.  Please just do a PR for that?

**Comment by sanhue0302** [comment]
remove import of GDataBooks from GData
it seems be removed from project
while i compiled the code it would cause error

File: Source/BaseClasses/GDataServiceBase.h [hunk 1]
```diff
@@ -21,8 +21,8 @@
   #define GDataUploadFetcherClass GTMSessionUploadFetcher
   #define GDataUploadFetcherClassStr @"GTMSessionUploadFetcher"
 
-  #define kGDataFetcherStatusUnauthorized kGTMSessionFetcherStatusUnauthorized
-  #define kGDataFetcherStatusForbidden kGTMSessionFetcherStatusForbidden
+  #define kGDataFetcherStatusUnauthorized GTMSessionFetcherStatusUnauthorized
+  #define kGDataFetcherStatusForbidden GTMSessionFetcherStatusForbidden
```
Comment (thomasvl) [inline Source/BaseClasses/GDataServiceBase.h]: Wasn't this already landed when https://github.com/google/gdata-objectivec-client/pull/193 was merged in?

Comment (sanhue0302) [inline Source/BaseClasses/GDataServiceBase.h]: ok~ but my question is how to create a pull request without #193 ?
should i create a new pull request to merge you master change on my fork?

Comment (thomasvl) [inline Source/BaseClasses/GDataServiceBase.h]: If you sync back with the main repo, it should have this change, and then this PR shouldn't include it.

Comment (sanhue0302) [inline Source/BaseClasses/GDataServiceBase.h]: sorry~
i still not understand what you mean "sync back with the main repo"
i merge master change to my fork, then this pull request increase a commit
Can you give me some tip about sync with main repo?

Comment (thomasvl) [inline Source/BaseClasses/GDataServiceBase.h]: You seen you update our fork against the master so this change doesn't show up; as it has already landed.

File: Source/Networking/GDataProgressMonitorInputStream.h [hunk 2]
```diff
@@ -47,11 +47,11 @@
   unsigned long long dataSize_;     // size of data in the source
   unsigned long long numBytesRead_; // bytes read from the input stream so far
 
-  __weak id monitorDelegate_;
+  id monitorDelegate_; //__weak
```
Comment (thomasvl) [inline Source/Networking/GDataProgressMonitorInputStream.h]: Actually, you likely don't even need the this file in your build.

**Comment by thomasvl** [comment]
https://github.com/google/gdata-objectivec-client/pull/195  I think batch fixed a bunch of these issues.

**Comment by sanhue0302** [comment]
thank you
i think i did it in a wrong way
i should not merge my change first
and i think i should delete my fork and fork from master again to keep my fork clear
i will delete this pull request later
thanks again

**Pull request closed without merging**